### ISSUE TITLE
Support for unreal engine version 4.23 because of HTML exporting

### ISSUE THREAD

**Issue by Dlacho1** [JanSeliv/CustomShapeButton#11]
Hello, I really really really like your plugin. It is a game changer for me. Can You please somehow add support to version 4.23 because I am working on one project that needs to go to the web and 4.23 is the last version supported for HTML exporting. I was trying to do it my self by updating your code but constantly getting some errors. I am also not good with C++ coding so that is also a reason why.
I thank You in advance and greet You warmly :)

**Comment by JanSeliv** [comment]
Hello, thank you for your interest in this plugin and your kind words:)

I developed this plugin specifically for Unreal Engine 5 because, at that time, all available solutions were only functional on UE4: I never tried my code on UE4. 

Unreal Engine 4.23 was released in 2019, and the engine has changed significantly since then. It may be very challenging to adapt it for older UE4 versions. Even if I manage to do so, there's no guarantee it will work properly.

Have you tried any existing solutions from other developers? There are plenty of them exactly for older UE4 versions. For instance, next one is for UE4.19, which should also work with UE4.23: [link](https://forums.unrealengine.com/t/free-custom-shaped-buttons-custom-hit-box-for-any-button/74724/30).
